src/modules/render.js

```diff
@@ -18,18 +26,26 @@ const clickEvents = () => {
 const renderShows = (async () => {
   let render;
   const shows = await getvShow();
+  const likes = await getlikes();
+  let likeCount = 0;
   shows.forEach((e) => {
+    likes.forEach((eachLikes) => {
+      if (String(eachLikes.item_id) === String(e.id)) {
+        likeCount = eachLikes.likes;
+      }
+    });
     render += `
     <li class="show"  id=${e.id}><img src=${e.image.medium} alt="movie-poster" class="movie-poster">
     <h3 class="show-name">${e.name}</h3>
     <div class="show-btns">
-    <i class="fa-solid fa-heart">
+    <i class="fa-solid fa-heart like-btn">
     </i>
-    <p></p>
+    <p> ${likeCount}</p>
     <i class="fa-solid fa-comment"></i>
     </div>
     </li>
     `;
+    likeCount = 0;
```

Comment:
I noticed that you are reassigning a value to `likeCount` here. I think that isn't needed

Reply:
that's to reset the value back to 0